activestorage/test/optimizer/jpeg_optimizer_test.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require "test_helper"
+require "database/setup"
+
+require "active_storage/optimizer/web_image_optimizer"
+
+class ActiveStorage::Optimizer::JpegOptimizerTest < ActiveSupport::TestCase
+  test "transforming with vips" do
+    optimize_variants_with(:vips) do
+      optimizer = ActiveStorage::Optimizer::JpegOptimizer.new(nil)
+
+      assert_equal "jpg", optimizer.transformations[:format]
+      assert_equal 85, optimizer.transformations[:saver][:quality]
+    end
+  end
+
+  test "transforming with image magick" do
+    optimize_variants_with(:mini_magick) do
```

Comment:
Oh I thought we were deprecating mini magick support, but it seems we aren't. I'm a little less enthused about having to maintain two variants here.

Reply:
Yeah. Vips was made the new default, but mini_magick was kept. 

The difference between mini magick and vips here is small though. It's mostly the optimization options passed in. Eg: `interlace: true` for vips vs `interlace: "JPEG"` for mini_magick.

We could also remove both the JPEG and PNG optimizers, and only leave WebImageOptimizer. This way the feature itself is preserved (allowing devs to change the default transformations), without having to maintain two variants.